CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 - Support stored procedure register with packages given as Python modules.
 - Added support for structured type schema parsing.
+- Removed import of modin.pandas.Index referring to native pandas index, instead having it import our own class that is currently a wrapper to native pandas index
```

Comment:
@sfc-gh-samdesai1 Let's move this to PANDAS_CHANGELOG, and agreed with @sfc-gh-vbudati with simplifying this line to say the new class is a wrapper class

Reply:
I changed it, lmk if it is good